xml/System.Linq/Enumerable.xml

```diff
@@ -11993,9 +11983,7 @@ If `count` is not a positive number, this method returns an identical copy of th
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- This method returns zero if `source` contains no elements.
-
- The result does not include values that are `null`.
+ Items in `source` that are null are excluded from the computation of the sum. This method never returns `null`.
```

Comment:
Any particular reason why this remark uses different wording than the ones above?

Reply:
Right, my point is that it's not applied to all methods consistently. Shouldn't it be the same everywhere?

Reply:
I see, I understand now,  thanks! 
I was thinking that the wordings only applicable to Sum that takes nullable parameters such as `int?` 
Ok, I'm going to apply to all methods as well today. 🙂

Reply:
Are some of these overloads not taking nullable parameters? I haven't checked, but I would assume they do because the remarks already make reference to `null` values.

Reply:
> Are some of these overloads not taking nullable parameters? I haven't checked, but I would assume they do because the remarks already make reference to `null` values.

AFAIK yes. For example:
https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.sum?view=net-6.0#system-linq-enumerable-sum(system-collections-generic-ienumerable((system-decimal)))

and looking at the remark, I think it doesn't have reference to `null` values:

![image](https://user-images.githubusercontent.com/8773147/190429819-e9cd4488-9120-416d-a542-feb548197496.png)

CMIIW, I think I need to ask your confirmation about this, because going back to my original understanding before, the update that I have applied should only matter for sum that takes IEnumerable<T> that takes T nullable such as `int?` or `double?`, like below overloads:

![image](https://user-images.githubusercontent.com/8773147/190431368-05afb23a-6972-4b18-9638-802aef61e44c.png)

Reply:
All remark sections that this PR updates originally contain this comment:

> The result does not include values that are `null`.

Presumably they're all nullable variants? If so then the same change should be applied everywhere.

Reply:
> Presumably they're all nullable variants? If so then the same change should be applied everywhere.

Yes, I've only updated all of the remarks that have that sentence and as long as the IEnumerable parameter has with T of nullable primitive number types such as `int?`, `long?`, `Double?`.

Ok, let me check again and ensure that I'm going to update all of the remarks section of any Sum that has met the above requirements.